jmxtrans-core/src/test/java/com/googlecode/jmxtrans/model/ServerTests.java

```diff
@@ -85,6 +85,50 @@ public void testEquals() {
 	}
 
 	@Test
+	public void testServerVariableHandling() {
+		// we add some variables to the System.properties list 
+		String alias = "somealias";
+		String port = "1234";
+		String host = "localhost.local";
+		String username = "acme";
+		String password = "password";
+		String url = "service:jmx:remoting-jms://amce.local:1234";
+		
+		System.setProperty("myalias", alias);
+		System.setProperty("myport", port);
+		System.setProperty("myhost", host);
+		System.setProperty("myusername",username);
+		System.setProperty("mypassword", password);
+		System.setProperty("myurl", url);
+		
+		Server s1 = Server.builder()
```

Comment:
To improve readability, we could call this `Server` "serverFromSystemProperties" and `s2` could be called "serverFromDirectParameters", or something similar.